bin/nrnivmodl.in

```diff
@@ -68,6 +70,16 @@ while [ "$1" ] ; do
         UserCOREFLAGS+=(-l "${2}")
         shift
         shift;;
+    -nmodl)
+        UserNMODLBIN="$2"
+        UserNMODLFLAGS="--neuron$UserNMODLFLAGS"
```

Comment:
There might be a missing space here, then again the variable is empty.

Reply:
I don't think an extra space is needed here since the space should always be added when initializing `UserNMODLFLAGS` underneath. I added an extra one here just for safety though in case the way we initialize `UserNMODLFLAGS` changes